examples/speed_tester/speed_tester.c

```diff
@@ -93,6 +93,8 @@ static uint8_t keep_running = 1;
 static uint16_t packet_size = ETHER_HDR_LEN;
 static uint8_t d_addr_bytes[ETHER_ADDR_LEN];
 
+/*  track the -c option to see if it has been filled */
+static uint8_t if_c = 0;
```

Comment:
I think we should rename this variable as this name is not the most clear way to represent that the flag to set packet count is enabled.

How about `use_custom_pkt_count` ?

Reply:
Sounds good. I will update.